Formula/qt.rb

```diff
@@ -27,6 +27,14 @@ class Qt < Formula
   uses_from_macos "flex"
   uses_from_macos "sqlite"
 
+  # Fix build on Linux when the build system has AVX2
+  # Patch submitted at https://codereview.qt-project.org/c/qt/qt3d/+/303993
+  patch do
+    url "https://codereview.qt-project.org/gitweb?p=qt/qt3d.git;a=patch;h=b456a7d47a36dc3429a5e7bac7665b12d257efea"
+    sha256 "e47071f5feb6f24958b3670d83071502fe87243456b29fdc731c6eba677d9a59"
+    directory "qt3d"
+  end
+
```

Comment:
@MikeMcQuaid The patch has been merged in the branch 5.15 of qt3d. It will be in qt 5.15.1. See:
https://codereview.qt-project.org/c/qt%2Fqt3d/+/312133

Should I create a new PR to document that fact directly in the formula? Or can you commit a comment yourself?

Reply:
Great work! It'll likely just get removed when it fails to apply in the Qt 5.15.1 bump 😎.